.github/scripts/spellcheck.words

```diff
@@ -484,6 +484,7 @@ Monnerat
 monospace
```

Comment:
for the commit message (not related to this here), I think you can remove:

```
An HTTP test server indicating whether MPTCP (Multipath TCP) is utilized
is now available at the following address: http://test.multipath-tcp.org:5000/.
```

But place it in the PR description: the service might not be available after the PR, maybe best not to keep this in the Git history.